windows/windows_update_install/install_msu_file.yml

```diff
@@ -1,10 +1,24 @@
 # Copyright 2023-2024 VMware, Inc.
 # SPDX-License-Identifier: BSD-2-Clause
 ---
+# Parameters:
+#   msu_file_path: the .msu file path in guest OS.
+#
+- name: "Get the .msu file basename"
+  include_tasks: ../utils/win_execute_cmd.yml
+  vars:
+    win_powershell_cmd: "(Get-Item {{ msu_file_path }}).BaseName"
+
+- name: "Set the .msu file basename"
+  ansible.builtin.set_fact:
+    msu_file_basename: "{{ win_powershell_cmd_output.stdout_lines[0] }}"
+  when:
+    - win_powershell_cmd_output.stdout_lines is defined
+    - win_powershell_cmd_output.stdout_lines | length == 1
```

Comment:
With `when` conditions, `msu_file_basename` might not be set when it doesn't meet the conditions. Then below tasks will raise exception.

Reply:
Will add assert to test if msu_file_basenname is set successfully. Thanks.